platforms/aws/giantnetes/main.tf

```diff
@@ -119,15 +119,35 @@ locals {
   }
 }
 
-# Generate ignition config.
-data "gotemplate" "bastion" {
-  template = "${path.module}/../../../templates/bastion.yaml.tmpl"
-  data     = "${jsonencode(local.ignition_data)}"
+# Render files before referencing them in main state
+# It should be applied before main apply
+# terraform apply -target="module.ignition" ./ 
+# The limitation comes from the fact, that terraform functions
+# do not participate in calculation of the graph state
+module "ignition" {
+  source = "../../../modules/ignition"
+
+  ignition_data = "${local.ignition_data}"
+}
+
+# List with base64 encoded files, injected into main templates
+locals {
+  Files = {
+      "conf/hardening.conf" = filebase64("../../../files/conf/hardening.conf")
```

Comment:
Not stopper, but can we use ${path.module} instead of ../../../ ?
Seems cleaner and we protect us against any modification in the code structure